ci/lint.sh

```diff
@@ -7,7 +7,7 @@ source "${BASH_SOURCE[0]%/*}/_shlib.sh"
 
 main() {
     # rustflags are present because of: https://github.com/rust-lang/rust-clippy/issues/5749
-    runt env RUSTFLAGS="-Dwarnings" cargo clippy --color=always -- -D warnings
+    runt env RUSTFLAGS="-Dwarnings" cargo clippy --color=always -- -D warnings -A clippy::manual-non-exhaustive
```

Comment:
I had to add this because I'm getting this error:
```
error: this seems like a manual implementation of the non-exhaustive pattern
  --> src/datetime.rs:38:1
   |
38 |   pub enum SecondsFormat {
   |   ^---------------------
   |   |
   |  _help: add the attribute: `#[non_exhaustive] pub enum SecondsFormat`
   | |
39 | |     /// Format whole seconds only, with no decimal point nor subseconds.
40 | |     Secs,
41 | |
...  |
61 | |     __NonExhaustive,
62 | | }
   | |_^
   |
   = note: `-D clippy::manual-non-exhaustive` implied by `-D warnings`
help: remove this variant
  --> src/datetime.rs:61:5
   |
61 |     __NonExhaustive,
   |     ^^^^^^^^^^^^^^^
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#manual_non_exhaustive

error: this seems like a manual implementation of the non-exhaustive pattern
   --> src/format/parsed.rs:27:1
    |
27  |   pub struct Parsed {
    |   ^----------------
    |   |
    |  _help: add the attribute: `#[non_exhaustive] pub struct Parsed`
    | |
28  | |     /// Year.
29  | |     ///
30  | |     /// This can be negative unlike [`year_div_100`](#structfield.year_div_100)
...   |
110 | |     _dummy: (),
111 | | }
    | |_^
    |
help: remove this field
   --> src/format/parsed.rs:110:5
    |
110 |     _dummy: (),
    |     ^^^^^^^^^^
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#manual_non_exhaustive
```

This is not related to my change. It's just a new rule of clippy in nightly (probably added yesterday).